src/user/components/change_password_form.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const form_fields = [ 'new_password', 'confirm_password' ]
+
+export default ({fields, handleSubmit}) =>
+  <form onSubmit={handleSubmit}>
+    <div className='change-password-inputs'>
+      {form_fields.map(input_maker(fields))}
+    </div>
+    <button className='change-password-button' type='submit'>Submit</button>
+  </form>
+
+
+const input_maker = fields => field =>
```

Comment:
I thought that it didn't quite match up due to having to set the type to password, not having edit or view mode and a few other things. Also as it was just two fields I thought it might be easier to just have some custom ones. Should I reuse `Field.js` anyway? @rjmk